A New CIV Class Ship: The PB Bratan!

Rubeebeebee opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
<!-- What did you change? -->
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->
I've added the PB Bratan! a CIV class ship thats part nightclub, part cafeteria, part hydroponics, part drug lab and smuggling ship, all in one! 

It's practically a moving POI, and it drives like one. 

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
It's got zero ship weaponry and absolutely abysmal speed and turning. 
It attempts to compensate for it by having a decent- for a civ ship- crew armory. 

they have a lethal and non lethal armory

Non Lethal:
3x Disablers
2x kammerers (beanbag)
(two boxes of beanbag shells)

3x Stunbatons 
2x Baseball bats

Lethals:
1x AKM
3x Gestio
1x Mosin 

(3 mags per gun)

3x Truncheons
2x Breaching Hammers

for armory, they have 2x armor vest and combat helmets, as well as 2x riot armor and 2x riot helmet, + 2x riot shields

## How to test
<!-- Describe the way it can be tested -->
load the .yml file in the IDE, should work! 

## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc). 
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->
<img width="899" height="803" alt="bratan live" src="https://github.com/user-attachments/assets/5de9387b-5b09-4371-b017-71f0b3feb915" />
<img width="854" height="800" alt="bratan" src="https://github.com/user-attachments/assets/d787d661-9231-4ef8-8464-05b0736275da" />
<img width="973" height="813" alt="bratan north" src="https://github.com/user-attachments/assets/c211a7d8-d290-4b63-896a-86e1ccb4b5ea" />
<img width="932" height="822" alt="bratan south" src="https://github.com/user-attachments/assets/b2334229-f0a7-4e63-8982-16cfc3004067" />
<img width="695" height="498" alt="bratan VIP room" src="https://github.com/user-attachments/assets/e32c3562-5379-4d35-bc18-4e433be4b33f" />
<img width="428" height="487" alt="vault room 2" src="https://github.com/user-attachments/assets/3c770856-1854-4840-87ee-ffdc513f8621" />



## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.
- [X] I can confirm this PR contains no AI-generated content, and did not use any AI-generated content.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->

:cl:
- add: the PB Bratan, a CIV service megaship with a hidden drug lab

Rubeebeebee committed:
I want to add another gas mining ship- so here is the HES-Phaeron, a tanker ship with a huge tank, but you'll have to carry either one filtered gas, or unfiltered gas.

Rubeebeebee committed:
i added decals to the twilight

Rubeebeebee committed:
Merge branch 'Monolith-Station:main' into main

Rubeebeebee committed:
Merge branch 'Monolith-Station:main' into main

Rubeebeebee committed:
changed the description to include a warning for non-cult members piloting the arkansaw, since its currently causing problems in the serv

Rubeebeebee committed:
Adding a new ship, the bratan

monolith8319 committed:
chore: automatically update REUSE headers

UnicornOnLSD commented:
I assume PB stands for Paycheck Bravta, if they made it you could instead give it the PBS title (for Paycheck Bravta Shipyards)

Rubeebeebee committed:
Merge branch 'main' into Bratan-Branch

Rubeebeebee commented:
> I assume PB stands for Paycheck Bravta, if they made it you could instead give it the PBS title (for Paycheck Bravta Shipyards)

as far as i can tell from the lore, PB doesnt delve into ship making, but they can probably contract other shipyards to build their designs for them, hence why the company name is just PB

UnicornOnLSD commented:
In that case it could either harbor the name of the company who made it example: SHS (Steel Hammer Shipyards), a composed tag SHS-PB or perhaps SHS [ship name] (PB) to signal it's a variation made by/for the PB

OneZerooo0 commented:
Steel Hammer Manufacturing makes the actual hulls, but the tags can stay as that of the source company (assuming they’re the ones who designed it)

Rubeebeebee committed:
Merge branch 'Monolith-Station:main' into Bratan-Branch

UnicornOnLSD commented:
>Steel Hammer Manufacturing makes the actual hulls

Makes some  of the hulls in the sectorr not all I believe
It's up to the author I'm just adding my pinch of salt to the lore friendly naming TM

Rubeebeebee committed:
Delete Resources/Prototypes/_Mono/Shipyard/Expedition/arkansaw.yml

the arkansaw is not supposed to be here.

OneZerooo0 commented:
I’m just using Steel Hammer Manufacturing largely as an example, I didn’t mean to force it on lol - just that I think tags should keep respect design origin

UnicornOnLSD commented:
> tags should keep respect design origin

Totally agreed, it's up to the person who made the ship in the end ! Since we don't have a formal naming convention.
Be it SHS-BR or something else taht works

Rubeebeebee commented:
I FUCKIN FORGOR TO MAKE WARP AND LATEJOIN SPAWN POINTS

Rubeebeebee committed:
added warm points + tweaked some decals

monolith8319 committed:
chore: automatically update REUSE headers

Rubeebeebee commented:
> > tags should keep respect design origin
> 
> Totally agreed, it's up to the person who made the ship in the end ! Since we don't have a formal naming convention. Be it SHS-BR or something else taht works

swanky ideas! ill make sure to keep these in mind for next time

Rubeebeebee committed:
updated the decals, added a fax and holopad, and changed the armory doors to a glass airlock, so the shutter can close/open

monolith8319 committed:
chore: automatically update REUSE headers

Rubeebeebee commented:
Updated the decals on the ship, added a fax and holopad to facilitate communications between ships, and ive changed the armory door to a glass airlock so the shutters can close and open

ive also added a cutter machine for the stage, so people can just re-tile the stage to fit whatever they want
<img width="610" height="873" alt="bratan decals3" src="https://github.com/user-attachments/assets/6b8253fe-2456-4779-bafc-af93550069f5" />
<img width="892" height="223" alt="bratan decals2" src="https://github.com/user-attachments/assets/30afbedd-98e2-4a2c-9d6a-75a570606688" />
<img width="504" height="559" alt="bratan decals1" src="https://github.com/user-attachments/assets/024628f3-1361-4d48-9405-ab779ffe53aa" />

Rubeebeebee committed:
Merge branch 'Monolith-Station:main' into Bratan-Branch

OneZerooo0 reviewed (changes requested):
Armory is a bit much for a Civilian vessel - Tone it down please. 
Decals are fine, could use a bit more flair in the food eating area.
Power is not sufficient, please use a bigger power generator like an AME.
It is honestly a bit ridiculous in how fast it moves - reduce the number of thrusters please.
Otherwise seems nice.

pheenty commented:
Bratan is a bad name considering the company itself is Bratva. Bratan in Russian is just a way of saying "brother". Usually in a meaning of a friend or, in our context, a partner in crime, gang member. Really it's just "bro".
Bratva on its turn is literally "a set of bratans". So, well, calling a ship Bratan is both lazy and confusing (what if Bratva member calls other Bratva members bratans?). 

If you want to go with something Bratva-themed I'd suggest Malina (yes, it means raspberry), it's a criminal slang for a hideout, or Krysha (roof), a slang for both a house and illegal (mafia or smth) protection: A kryshuet B – A protects B.

UnicornOnLSD commented:
their ship their name if it's cool sounding and fits the vibe it's fine

pheenty commented:
> their ship their name if it's cool sounding and fits the vibe it's fine

Yeah Idrc I'm just saying that this is the same as calling a TSF ship "Colonel"

UnicornOnLSD commented:
> > their ship their name if it's cool sounding and fits the vibe it's fine
> 
> Yeah Idrc I'm just saying that this is the same as calling a TSF ship "Colonel"
faair.

OneZerooo0 commented:
You also need to add a station records computer

Rubeebeebee committed:
updated the dining room decals, added an AME for power generation, and adjusted armory to be less lethal

Rubeebeebee committed:
Merge branch 'Bratan-Branch' of https://github.com/Rubeebeebee/Monolith into Bratan-Branch

monolith8319 committed:
chore: automatically update REUSE headers

Rubeebeebee commented:
> Armory is a bit much for a Civilian vessel - Tone it down please. Decals are fine, could use a bit more flair in the food eating area. Power is not sufficient, please use a bigger power generator like an AME. It is honestly a bit ridiculous in how fast it moves - reduce the number of thrusters please. Otherwise seems nice.

alright! added an AME for power generation, removed the plasma generator because its frankly not needed anymore. 

removed some thrusters, 2 backwards thrusters, and 4 forward thrusters.

fixed some lattices not being vacuumed 

adjusted the armory to contain 3x disablers, 3x stun batons, and for the armory on the back- 2x kammerers, 1x double barrel shotgun (all loaded with beanbags), 3x baseball bats,  2x beanbag boxes, and 1x buckshot box

also added the stations record console. 

tweaked the diner to have more flair!
<img width="454" height="476" alt="bratan station records" src="https://github.com/user-attachments/assets/e1518ced-dc04-46e8-80c0-2577d57718ed" />
<img width="635" height="833" alt="bratan armory" src="https://github.com/user-attachments/assets/ef4a7c72-e11b-4a3f-b9fb-51ad43945787" />
<img width="716" height="680" alt="bratan diner" src="https://github.com/user-attachments/assets/05d2184b-406f-4e26-a885-f2483332c982" />
<img width="624" height="753" alt="bratan ame" src="https://github.com/user-attachments/assets/2d8afafd-3d0c-4293-b805-4ae123c9cdd0" />

OneZerooo0 commented:
Rotate your AME corners and it’s probably fine for approval

Rubeebeebee commented:
> Rotate your AME corners and it’s probably fine for approval

like the AME blocks themselves? or the walls containing the AME?

Rubeebeebee committed:
rotated the AME

Rubeebeebee committed:
Merge branch 'Bratan-Branch' of https://github.com/Rubeebeebee/Monolith into Bratan-Branch

Rubeebeebee commented:
> Rotate your AME corners and it’s probably fine for approval

there we go! i have rotated the AME corners!

monolith8319 committed:
chore: automatically update REUSE headers

OneZerooo0 reviewed (changes requested):
Don't remove the arkansaw please

Rubeebeebee commented:
> Don't remove the arkansaw please

im trying not to! hold on, ill make a new branch because i cant seem to delete the file that does it

Rubeebeebee commented:
@OneZerooo0 

Ive made a new branch for the updated bratan! (because i cant figure out how to remove the file that removes the arkansaw)

https://github.com/Monolith-Station/Monolith/pull/1641
